traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -34,29 +34,30 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		return nil, nil, err
 	}
 	return []Route{
-		{1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring", wrapHeaders(wrapAuth(monitoringHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},
-		{1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring.json", wrapHeaders(wrapAuth(monitoringHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},
+		{1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring", wrapHeaders(wrapAuth(monitoringHandler(rd.Monitoring), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},
+		{1.2, http.MethodGet, "cdns/{cdn}/configs/monitoring.json", wrapHeaders(wrapAuth(monitoringHandler(rd.Monitoring), d.Insecure, d.TOSecret, rd.PrivLevelStmt, MonitoringPrivLevel))},
 	}, rootHandler(d), nil
 }
 
 type RouteData struct {
 	PrivLevelStmt *sql.Stmt
+	Monitoring    *MonitoringData
```

Comment:
I think this is to specific at this level.  So, the developer will have to implement this for every single route?